lti_consumer/plugin/compat.py

```diff
@@ -110,7 +110,10 @@ def get_course_by_id(course_key):
     Import and run `get_course_by_id` from LMS
     """
     # pylint: disable=import-error,import-outside-toplevel
-    from lms.djangoapps.courseware.courses import get_course_by_id as lms_get_course_by_id
+    try:
+        from openedx.core.lib.courses import get_course_by_id as lms_get_course_by_id
+    except ImportError:
+        from lms.djangoapps.courseware.courses import get_course_by_id as lms_get_course_by_id
```

Comment:
Technically, this would still work, even after the move in `edx-platform` because that LMS module also imports the function anyway (from `openedx.core`), since it's used by another function in that LMS module.